backend/app/crud/crud_content.py

```
@@ -0,0 +1,221 @@
+
+    Returns:
+        The newly created and refreshed ContentItem object.
+    """
+    db_content_item = ContentItem.model_validate(content_item_in) # Ensure it's a valid model instance
+    session.add(db_content_item)
+    session.commit()
+    session.refresh(db_content_item)
+    return db_content_item
+
+def update_content_item(
+    session: Session, db_content_item: ContentItem, content_item_in: Any 
```

Ruff (0.11.9)

Trailing whitespace

Remove trailing whitespace

(W291)